streetscapes/conf.py

```diff
@@ -0,0 +1,28 @@
+# --------------------------------------
+from decouple import AutoConfig
+
+# --------------------------------------
+from pathlib import Path
+
+# Setup
+# ==================================================
+ROOT_DIR = Path(__file__).parent.parent
+LOCAL_DIR = ROOT_DIR / "local"
+config = AutoConfig(ROOT_DIR)
+
+# Location of the Streetscapes data
+# ==================================================
+DATA_DIR = Path(
+    config("STREETSCAPES_DATA_PATH", LOCAL_DIR / "streetscapes-data")
+)
+DATA_DIR.mkdir(exist_ok=True, parents=True)
```

Comment:
We could replace this with checking for the csv files and throwing an error if they cannot be found.